cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -18,12 +18,10 @@ type volumeAPIOpsV1alpha1 struct {
 // volumeV1alpha1SpecificRequest is a http handler to handle HTTP
 // requests to a OpenEBS volume.
 func (s *HTTPServer) volumeV1alpha1SpecificRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
-
-	//fmt.Println("[DEBUG] processing v1alpha1", req.Method, "request")
-	glog.Infof("Received volume request: Method: '%s' Version: 'v1alpha1'", req.Method)
+	glog.Infof("cas template based volume request was received: method '%s'", req.Method)
```

Comment:
nit: cas --> CAS ?

Reply:
w.r.t logging, I am trying to make every word as small case. any best practices guide that I can follow. I thought it will be easier to consume etc by tools.